cmake/features.cmake

```diff
@@ -9,6 +9,8 @@ ov_option (ENABLE_PROXY "Proxy plugin for OpenVINO Runtime" ON)
 
 ie_dependent_option (ENABLE_INTEL_CPU "CPU plugin for OpenVINO Runtime" ON "RISCV64 OR X86 OR X86_64 OR AARCH64 OR ARM" OFF)
 
+ie_dependent_option (ENABLE_CPU_EXTENSIONS "Enable cpu extensions library in CPU plugin" ON "ENABLE_INTEL_CPU AND X86_64 AND LINUX" OFF)
```

Comment:
In MLAS, I meet a compilation error #19005  when using WebAssembly. Could you check whether this library compatible with WebAssembly build ? If not you should disable it.

Reply:
The WebAssembly can be compiled with current default setting.